### PR TITLE
PSMDB-1293 Added build from source instructions

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/install/index.md
	new file:   docs/install/source.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/install/source.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+    -  Build Percona Server for MongoDB from ``buildscripts/scons.py``.
+        
+        ```{.bash data-prompt="$"}
+        $ buildscripts/scons.py -j$(nproc --all) --jlink=2 --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib" install-mongod
+        ``` 
+
+       This command builds only the database. Other available targets for the
+       ``scons`` command are:  
+
```
Comment (igorsol) [inline docs/install/source.md:119]: all those targets must include "install-" prefix in v6.0.

Comment (igorsol) [reply docs/install/source.md:119]: install-mongod
install-mongos
and so on..

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/install/source.md (hunk 1 above)
Comment (nastena1606) [reply docs/install/source.md:119]: Updated
Added the build instructions for FIPS

**Review by igorsol — APPROVED** [review]

**Review by janwieremjewicz — APPROVED** [review]
LGTM

**Review by defbin — APPROVED** [review]
LGTM and useful 😸

File: docs/install/source.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+1. Change directory to ``percona-server-mongodb`` 
+   
+    ```{.bash data-prompt="$"}
+    $ cd percona-server-mongodb
+    ``` 
+
+2. Build Percona Server for MongoDB from ``buildscripts/scons.py``
+     
+    ```{.bash data-prompt="$"}
+    $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
```
Comment (sandraromanchenko) [inline docs/install/source.md]:
```suggestion
    $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos
```

Please remove target mongo, old shell is used just for autotests and customers don't need it

File: docs/install/source-fips.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+    ```{.bash data-prompt="$"}
+    $ cd percona-server-mongodb
+    ``` 
+
+2. Build Percona Server for MongoDB from ``buildscripts/scons.py``
+
+    === "Without FIPS mode"
+     
+        ```{.bash data-prompt="$"}
+        $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
```
Comment (sandraromanchenko) [inline docs/install/source-fips.md]:
```suggestion
        $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos
```

File: docs/install/source-fips.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+        ```  
+
+    === "With FIPS mode"
+     
+        ```{.bash data-prompt="$"}
+        $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup --enable-fipsmode CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
```
Comment (sandraromanchenko) [inline docs/install/source-fips.md]:
```suggestion
        $ buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup --enable-fipsmode CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Review by janwieremjewicz — APPROVED** [review]

**Pull request merged**